src/tests/test_utils/common_test_utils/include/common_test_utils/test_enums.hpp

```diff
@@ -60,7 +60,8 @@ enum EltwiseTypes {
     BITWISE_AND,
     BITWISE_NOT,
     BITWISE_OR,
-    BITWISE_XOR
+    BITWISE_XOR,
+    CONVERT_OP
```

Comment:
I don't like the idea to extend common enum on Convert operation type. It impacts all the components actually. 
Since we need it only for CPU EltwiseChainTest, lets extend its parameters to avoid impact on common defines. We can have new parameter like `bool addConvert` which will insert Convert operation before first Eltwise in the chain.